tests/integration/cloudformation/test_cloudformation_lambda.py

```diff
@@ -93,3 +95,43 @@ def test_update_lambda_inline_code(
     finally:
         # cleanup
         cleanup_stacks([stack_name])
+
+
+def test_lambda_w_dynamodb_event_filter(
+    cfn_client, dynamodb_client, is_stack_created, cleanup_stacks
+):
+    stack_name = f"stack-{short_uid()}"
+    function_name = f"test-fn-{short_uid()}"
+    table_name = f"ddb-tbl-{short_uid()}"
+    item_to_put = {"id": {"S": "test123"}}
+
+    try:
+        template_1 = jinja2.Template(load_template_raw("lambda_dynamodb_filtering.yaml")).render(
+            event_filter='{"eventName": ["INSERT"]}',
+            table_name=table_name,
+            function_name=function_name,
+        )
+
+        response = cfn_client.create_stack(
+            StackName=stack_name, TemplateBody=template_1, Capabilities=["CAPABILITY_IAM"]
+        )
+        stack_id = response["StackId"]
+        assert stack_id
+        wait_until(is_stack_created(stack_id))
+
+        # put item the first time: INSERT
+        dynamodb_client.put_item(TableName=table_name, Item=item_to_put)
+
+        def assert_single_lambda_call():
+            events = get_lambda_log_events(function_name)
```

Comment:
similar to the other test: we need to add the `logs_client` so retrieve the logs from AWS `get_lambda_log_events(function_name, logs_client=logs_client)`